src/lib/config/defaultConfig.js

```diff
@@ -175,6 +175,10 @@ export const RECORDS_CONFIG = {
         text: 'To my office',
         iconClass: 'dolly',
       },
+      'SELF-CHECKOUT': {
```

Comment:
Honestly, I think that I would not have added the delivery method here, and a condition to filter it out.
It is a special one. I think that it is safer to have an extra config key, e.g. `deliveryMethodSelfCheckout: ....`